libraries/AC_WPNav/AC_Circle.cpp

```diff
@@ -98,7 +98,7 @@ void AC_Circle::init()
 /// set_circle_rate - set circle rate in degrees per second
 void AC_Circle::set_rate(float deg_per_sec)
 {
-    if (!is_equal(deg_per_sec,_rate)) {
+    if (!is_equal(deg_per_sec,(float)_rate)) {
```

Comment:
because since is_equal() is a template function now, the compiler would try to use an `is_equal(float, AP_Float<...>)` instead of implicitly casting to float. However we only instantiate the template function for the basic types in order to avoid creating a handful of small functions and increasing the binary size.